plugins/GWSplits/GWSplits.cpp

```diff
@@ -650,16 +801,48 @@ void GWSplits::DrawSettings()
 
     if (GW::Map::GetInstanceType() == GW::Constants::InstanceType::Loading) return;
 
+    ImVec2 pos(0, 0);
+    if (const auto window = ImGui::FindWindowByName(Name())) {
+        pos = window->Pos;
+    }
+
+    if (ImGui::DragFloat2("Position", reinterpret_cast<float*>(&pos), 1.0f, 0.0f, 0.0f, "%.0f")) {
+        ImGui::SetWindowPos(Name(), pos);
+    }
     ImGui::Checkbox("Lock Position", &lock_move);
     ImGui::SameLine();
     ImGui::Checkbox("Lock Size", &lock_size);
-    ImGui::SameLine();
+
+    ImGui::Text("Display: ");
+    ImGui::Indent();
     ImGui::PushItemWidth(150.f);
+    ImGui::DragInt("Total splits", &totalSplits, 1, 0, 0);
```

Comment:
```suggestion
    ImGui::DragInt("Displayed splits", &totalSplits, 1, 0, 0);
```
I wasnt sure what "total splits" meant when seein it at first, maybe this is more clear?